<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>momo_fighter</class>
 <widget class="QWidget" name="momo_fighter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="me_name">
     <property name="text">
      <string>空条承太郎</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>568</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" rowspan="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>568</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLabel" name="enemy_name">
     <property name="text">
      <string>迪奥·布兰度</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="me_heamal_strand">
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	text-align:center;
}
QProgressBar::chunk{
	background-color:rgb(255, 0, 0);
}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QProgressBar" name="enemy_heamal_strand">
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	text-align:center;
}
QProgressBar::chunk{
	background-color:rgb(255, 0, 0);
}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListView" name="me_property_list"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="me_pic">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="enemy_pic">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QListView" name="enemy_property_list"/>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLabel" name="action_label">
     <property name="text">
      <string>请选择行动</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>1151</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QPushButton" name="escape_button">
          <property name="text">
           <string>逃跑</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="common_attack_button">
          <property name="text">
           <string>普通攻击</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="tools_button">
          <property name="text">
           <string>道具</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="skill_button">
          <property name="text">
           <string>技能</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QListWidget" name="skill_listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1161</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>1040</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QListWidget" name="tools_listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1161</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1040</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
